contracts/PartyToken.sol

```diff
@@ -5,18 +5,25 @@ pragma solidity 0.8.5;
 import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
 import {ERC20Permit} from "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
 import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
 
 /*
 Party Token
 by Anna Carroll
 */
-contract PartyToken is ERC20VotesComp, Pausable {
+contract PartyToken is ERC20VotesComp {
+    // ============ Events ==============
+    
+    event Unpaused(address account);
+
     // ============ Immutables ============
 
     address public immutable partyDAOMultisig;
     address public immutable deprecationContract;
 
+    // ======== State =========
+
+    bool lockupPeriod = true;
```

Comment:
bikeshedding, but i would call it `lockupIsActive`, as `lockupPeriod` sounds more like i'll get a uint256 number of blocks for which lockup is active